Sources/EdgeNetworkHandlers/EdgeHitProcessor.swift

```diff
@@ -74,12 +77,7 @@ class EdgeHitProcessor: HitProcessing {
 
         // Build Request object
         let requestBuilder = RequestBuilder()
-        if let ecid = identityState[EdgeConstants.SharedState.Identity.ECID] as? String {
-            requestBuilder.experienceCloudId = ecid
-        } else {
-            // This is not expected to happen. Continue without ECID
-            Log.warning(label: LOG_TAG, "processHit - An unexpected error has occurred, ECID is nil.")
-        }
+        requestBuilder.xdmPayloads = [AnyCodable.from(dictionary: identityState) ?? [:]]
```

Comment:
We might need to extract the identityMap here as well. If Identity XDM Shared States gets updated with new fields in the future and we shouldn't include those in the Edge requests, this will break the network requests for old implementations of IdentityEdge.

Reply:
If this is the case, then this breaks the contract that we have laid out in previous conversations of the Edge extension being able to blindly attach the IdentityMap.

If that is required we can change later, but we shouldn't be pulling out the IdentityMap then grooming it before the network request is sent out. I thought the point was that the map would be in the correct format and we would not need to make any modifications to it before sending it.

Reply:
I am refering to the case where com.adobe.identityedge XDM shared state becomes:

Reply:
I find it a bit odd that we must parse the Identity shared state in the Edge extension, but I've updated the PR to only include the IdentityMap.
https://github.com/adobe/aepsdk-edge-ios/pull/165/commits/ecd2dcbb9872357a2ee3e591262d97b1ee203a68